packages/react/src/components/Chat/ChatMessage.tsx

```diff
@@ -83,7 +84,7 @@ export interface ChatMessageProps
   onFocus?: ComponentEventHandler<ChatMessageProps>
 
   /** Reaction group applied to the message. */
-  reactionGroup?: ShorthandValue
+  reactionGroup?: ShorthandValue<ReactionGroupProps>
```

Comment:
Shouldn't this be: `ShorthandValue<ReactionGroupProps> | ShorthandCollection<ReactionProps>`